core/src/main/resources/lib/form/select/select.js

```diff
@@ -5,9 +5,14 @@ function updateListBox(listBox,url,config) {
     config = object(config);
     var originalOnSuccess = config.onSuccess;
     var l = $(listBox);
-    var status = findFollowingTR(listBox, "validation-error-area").firstChild.nextSibling;
-    if (status.firstChild && status.firstChild.getAttribute('data-select-ajax-error')) {
-        status.innerHTML = "";
+    var status;
+    try {
+        status = findFollowingTR(listBox, "validation-error-area").firstChild.nextSibling;
+        if (status.firstChild && status.firstChild.getAttribute('data-select-ajax-error'))
+            status.innerHTML = "";
+    } 
+    catch(e) {
+        console.log(e);
```

Comment:
this doesn't really solve the problem, does it? just logs instead of `throw`ing?

Reply:
I suggest:

1) find out what scenario causes this problem
and
2) either extract out checks for `.firstChild` and `.nextSibling` or change a selector such that it fixes the root cause of the problem.

Reply:
Jenkins console error:-
![image](https://user-images.githubusercontent.com/12015596/51850538-f1ecf800-2347-11e9-92fb-81d6b53a914b.png)

jelly config file
![image](https://user-images.githubusercontent.com/12015596/51850579-0a5d1280-2348-11e9-9459-0b8680609b28.png)

So, this error is caused by updateListBox function.. Any suggestions on fixing this would be great.

Fyi... the same works totally fine on jenkins verion 2.7.3